tests/unit/test_bruteforce.cpp

```diff
@@ -934,32 +833,66 @@ TEST_F(BruteForceTest, brute_force_test_inf_score) {
             ASSERT_EQ(3, id);
         } else {
             ASSERT_TRUE(id == 2 || id == 4);
+            ASSERT_TRUE(std::isinf(score));
         }
     };
     runTopKSearchTest(index, "abcdefgh", k, verify_res);
     VecSimIndex_Free(index);
 }
 
-TEST_F(BruteForceTest, brute_force_remove_vector_after_replacing_block) {
-    size_t dim = 4;
-    size_t n = 2;
+TEST_F(InfTest, brute_force_test_inf_score_fp64) {
```

Comment:
will using those numbers on float test yield the same result? in other words, can we reuse the test?

Reply:
exp(500) itself inf for float (for doubles its ok) so it doesn't check the scenario where we insert a valid number and the distance result is out of range
If this wasn't the purpose of the test we can reuse it